.github/workflows/docker-image.yml

```diff
@@ -9,6 +9,8 @@ on:
 
 jobs:
   push_to_registry:
+    with:
```

Comment:
@prashantasdeveloper FYI: seems like we need this line with submodules in our actions

Reply:
I merged this and @polymath-eric I get `Property with is not allowed.`